RadarSDK/RadarLocationManager.m

```diff
@@ -329,23 +332,47 @@ - (void)updateTracking:(CLLocation *)location {
             }
         } else {
             [self stopUpdates];
-            [self removeGeofences];
+            [self removeMovingStoppedGeofence];
             [self.locationManager stopMonitoringVisits];
             [self.locationManager stopMonitoringSignificantLocationChanges];
         }
     });
 }
 
-- (void)replaceGeofence:(CLLocation *)location radius:(int)radius {
-    [self removeGeofences];
+- (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
+    for (int i = 0; i < geofences.count; i++) {
```

Comment:
We may want to clear out the old synced geofences here

Reply:
Yes, good call